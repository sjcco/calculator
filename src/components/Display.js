import PropTypes from 'prop-types';

const Display = props => {
  const { result, operation } = props;
  return (
    <div className="display skin">
      <p>{ operation }</p>
      <p>{ result }</p>
    </div>
  );
};

Display.propTypes = {
  result: PropTypes.string,
  operation: PropTypes.string,
};

Display.defaultProps = {
  result: '0',
  operation: '',
};

export default Display;
