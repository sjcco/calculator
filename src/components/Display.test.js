import React from 'react';
import renderer from 'react-test-renderer';
import Display from './Display';

test('render page correctly', () => {
  const component = renderer.create(<Display />).toJSON();
  expect(component).toMatchSnapshot();
});
