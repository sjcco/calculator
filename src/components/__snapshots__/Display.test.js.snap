// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render page correctly 1`] = `
<div
  className="display skin"
>
  <p>
    
  </p>
  <p>
    0
  </p>
</div>
`;
